api/apps/articles/model/retrieve.coffee

```diff
@@ -33,6 +33,7 @@ moment = require 'moment'
       query.biography_for_artist_id = ObjectId input.biography_for_artist_id
     query.featured_artwork_ids = ObjectId input.artwork_id if input.artwork_id
     query.tags = { $in: input.tags } if input.tags
+    query.internal_tags = { $in: input.internal_tags } if input.internal_tags
 
```

Comment:
Since the queries on vertical and sub_vertical only allow a single string to be queried (and not an array), we don't need to do anything custom for it. Of course this is subject to change depending on our convo on Friday! This method is for queries that need a little more support than just passing them through to mongo directly.